controllers/machinehealthcheck_controller.go

```diff
@@ -210,7 +210,7 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	sort.Strings(m.Status.Targets)
 
 	// health check all targets and reconcile mhc status
-	healthy, unhealthy, nextCheckTimes := r.healthCheckTargets(targets, logger, m.Spec.NodeStartupTimeout.Duration)
+	healthy, unhealthy, nextCheckTimes := r.healthCheckTargets(targets, logger, *m.Spec.NodeStartupTimeout)
```

Comment:
Should we check the pointer here?

Reply:
If we get a nil pointer, is it reasonable to default it to the same value we set in the default for the CRD? Will need a new const for that